cpp/tests/utilities/column_utilities.hpp

```diff
@@ -90,26 +99,14 @@ void print(cudf::column_view const& col, std:: ostream &os = std::cout, std::str
 template <typename T>
 std::pair<std::vector<T>, std::vector<bitmask_type>> to_host(column_view c) {
   std::vector<T> host_data;
-  std::vector<bitmask_type> host_bitmask;
-
-  auto col = column(c);
 
-  if (col.size() > 0) {
-    host_data.resize(col.size());
-    CUDA_TRY(cudaMemcpy(host_data.data(), col.view().head<T>(),
-                        col.size() * sizeof(T),
-                        cudaMemcpyDeviceToHost));
-  }
-
-  if (col.nullable()) {
-    size_t mask_allocation_bytes = bitmask_allocation_size_bytes(c.size());
-    host_bitmask.resize(mask_allocation_bytes);
-    size_t num_mask_elements = num_bitmask_words(c.size());
-    CUDA_TRY(cudaMemcpy(host_bitmask.data(), col.view().null_mask(),
-                        num_mask_elements * sizeof(bitmask_type),
+  if (c.size() > 0) {
+    host_data.resize(c.size());
+    CUDA_TRY(cudaMemcpy(host_data.data(), c.head<T>(), c.size() * sizeof(T),
                         cudaMemcpyDeviceToHost));
   }
 
+  std::vector<bitmask_type> host_bitmask = bitmask_to_host(c);
```

Comment:
IILE could be utilized here.
```cpp
  auto host_bitmask = bitmask_to_host(c);
  auto host_data = [&] () {
    if (c.size() > 0) {
      host_data.resize(c.size());
      CUDA_TRY(cudaMemcpy(host_data.data(), c.head<T>(), c.size() * sizeof(T),
                          cudaMemcpyDeviceToHost));
    } else 
      return std::vector<T>{};
  } ();
```

Reply:
This seems to add complexity to simple code.

Reply:
This is just the same change as above, removing `.resize` ~ more declarative.

Reply:
Surely this doesn't require a lambda.

Reply:
I kind of understand IILE now. I recall that I prefer using ternary operators instead of regular `if else` when conditionally assigning a value. So this IILE here is like a generalisation of ternary operators? By using this, you can use `auto` to declare such variables. But IDK if omitting a return statement from the first `if` block in favour of capturing the variable name itself makes it any cleaner. I'd just add the return statement.